admin/data/sql/migrations/143.do.create-check-window-with-status-view.sql

```diff
@@ -0,0 +1,9 @@
+CREATE VIEW [mtc_admin].vewCheckWindowWithStatus
+AS SELECT cw.urlSlug, cw.name, cw.isDeleted, CAST(
+    CASE
+        WHEN GETUTCDATE() < cw.adminStartDate THEN 'Inactive'
+        WHEN GETUTCDATE() >= cw.adminStartDate AND GETUTCDATE() <= cw.adminEndDate THEN 'Active'
+        ELSE 'Past'
+    END AS NVARCHAR(50)
+   ) AS [status]
+FROM mtc.mtc_admin.checkWindow cw
```

Comment:
Don't include the database name `mtc` please

Reply:
@jon-shipley good spot, updated.